### DISCUSSION TITLE
Routable maps avoids highways

### DISCUSSION THREAD

**Discussion by gerrievisagie** [roballey/OSM_Maps_For_Garmin#6] [Q&A]
First of all, this is an awesome maps set. Very detailed and accurate.

I have a Garmin overlander. I installed the New Zealand routable map onto a memory card on the device. Everything shows up, however when routing it appears to avoid highways? I checked the avoidances section of the navigation setup but nothing is avoided. 
Am I doing something wrong here?

**Comment by roballey** [comment]
Sorry I would not know why it avoids highways as I use these maps on various fitness devices for foot or bike based activities and do my route planning using brouter on a PC.  Just confirmed on an Edge 1050 that a route I would expect to give via the N1 motorway avoided it.
